pkg/globalconfig/global_config.go

```diff
@@ -308,6 +309,11 @@ func WriteGlobalConfig(config GlobalConfig) error {
 		return err
 	}
 
+	// Remove empty project list from global config
+	// In a future release it will be removed from the struct but
+	// for now it's needed to make upgrading easier.
+	cfgbytes = []byte(strings.Replace(string(cfgbytes), "project_info: {}", "", 1))
```

Comment:
This is to ensure we don't have an **empty** project list in global config. It felt quite hacky, but was better than keeping an empty list in the file which would lead to further confusion.